samples/nfc/writable_ndef_msg/src/ndef_file_m.c

```diff
@@ -28,10 +29,11 @@ static const u8_t m_url[] = /**< Default NDEF message: URL "nordicsemi.com". */
 	{'n', 'o', 'r', 'd', 'i', 'c', 's', 'e', 'm', 'i', '.', 'c', 'o', 'm'};
 
  /* Flash block size in bytes */
-#define NVS_SECTOR_SIZE    (DT_FLASH_WRITE_BLOCK_SIZE * 1024)
-#define NVS_SECTOR_COUNT   2
+#define NVS_SECTOR_SIZE  (DT_PROP(DT_CHOSEN(zephyr_flash), erase_block_size))
```

Comment:
Are you sure this change is safe? I know 4 * 1024 == 4096, but presumably the writeblock size as chosen for a reason? I'm also not sure why this isn't using the flash_area_get_sectors() API

Reply:
@smyczekkamil was the original author of this sample, but he doesn't work for Nordic anymore. Having said that I didn't notice incorrect sector size configuration since I was maintaining this code.

This change looks good to me and thanks for improving it.

Reply:
@mbolivar-nordic @kapi-no eras-block-size should be used here. Previous assignment works with hidden assumption that flash page contains 1k independent write blocks and was incorrect in generall.